TestBangazonAPI/BangazonAPITests.cs

```diff
@@ -0,0 +1,101 @@
+using Newtonsoft.Json;
+using System;
+using System.Net;
+using System.Net.Http;
+using System.Text;
+using System.Threading.Tasks;
+using Xunit;
+using BangazonAPI;
+using BangazonAPI.Models;
+namespace BangazonAPITests
+{
+    public class CustomerTests
+    {
+        [Fact]
+        public async Task GetCustomer_Success()
+        {
+
+            using (var client = new APIClientProvider().Client)
+            {
+                var response = await client.GetAsync("api/Customer");
+                response.EnsureSuccessStatusCode();
+
+            }
+        }
+        [Fact]
+        public async Task GetCustomers_Success()
+        {
+
+            using (var client = new APIClientProvider().Client)
+            {
+                var response = await client.GetAsync("api/Customer/5");
+                response.EnsureSuccessStatusCode();
+
+            }
+        }
+
+        [Fact]
+        public async Task GetCustomersq_Success()
+        {
+
```

Comment:
Please verify that you got a customer back that matches the `q` query string parameter.

Reply:
fixed